pkg/api/v1/schema.go

```diff
@@ -327,10 +339,49 @@ type RandomGenerator struct {
 
 func NewMinimalDomainSpec() *DomainSpec {
 	domain := DomainSpec{OS: OS{Type: OSType{OS: "hvm"}}, Type: "qemu"}
-	domain.Memory = Memory{Unit: "KiB", Value: 8192}
+	domain.Memory = Memory{
+		Allocated: 8192 * 1024,
+		Required:  8192 * 1024,
+		Maximum:   4 * 8192 * 1024,
+	}
 	domain.Devices = Devices{}
 	domain.Devices.Interfaces = []Interface{
 		{Type: "network", Source: InterfaceSource{Network: "default"}},
 	}
 	return &domain
 }
+
```

Comment:
Do not sure if it a good place for this method, because this file used for schema definition only

Reply:
yeah, it doesn't really belong in the schema.go file. I think this function would make sense to be a helper function in template.go file... at least for now.  It's possible some of this resource quota/limit functionality will live in it's own package.

Reply:
This is a pure derived value that currently depends only on data coming from the Domain. I can move it elsewhere of course, the template.go is probably the closest fit right now.